Core/Commands/SpawnWorldCommand.cs

```diff
@@ -16,7 +16,7 @@ public sealed class SpawnWorldCommand : ModCommand
 
 	public override void Action(CommandCaller caller, string input, string[] args)
 	{
-		SubworldSystem.Enter<EaterDomain>();
+		SubworldSystem.Enter<BrainDomain>();
```

Comment:
Personally just changing it and then being able to type `/newworld` without params is nice because I do it a *lot*. Even thought of making it `/nw` to make it even quicker. Since it's debug only, I don't think it matters much.